power_grid_model_c/power_grid_model/include/power_grid_model/three_phase_tensor.hpp

```diff
@@ -198,8 +199,9 @@ template <column_vector DerivedA> inline double max_val(Eigen::ArrayBase<Derived
 
 // function to sum rows of tensor
 template <rk2_tensor DerivedA> inline auto sum_row(Eigen::ArrayBase<DerivedA> const& m) { return m.rowwise().sum(); }
-// overload for double
+// overload for double and double complex
 inline double sum_row(double d) { return d; }
+inline DoubleComplex sum_row(DoubleComplex d) { return d; }
```

Comment:
maybe should be `const&`

```suggestion
inline DoubleComplex const& sum_row(DoubleComplex const& d) { return d; }
```

but i'm not sure cause it actually "computes" a result and if the user changes the input, this result should not change... of course, it would be the user's responsibility to use `auto` instead of `auto const&` anyways in that case, but... difficult. let's discuss